code/game/gamemodes/steal_items.dm

```diff
@@ -189,16 +212,18 @@
 		var/found_amount = 0
 		for(var/obj/I in all_items) //Check for items
 			if(istype(I, typepath))
+			
 				//Stealing the cheap autoinjector doesn't count
 				if(istype(I, /obj/item/weapon/reagent_containers/hypospray/autoinjector))
 					continue
+
 				if(istype(I,/obj/item/device/aicard))
 					var/obj/item/device/aicard/C = I
 					if(!C.contents.len)
 						continue //Stealing a card with no contents doesn't count
 					var/is_at_least_one_alive = 0
 					for(var/mob/living/silicon/ai/A in C)
-						if(A.stat != DEAD)
+						if(A.stat != 2)
 							is_at_least_one_alive++
 					if(!is_at_least_one_alive)
 						continue
```

Comment:
12 lines below this,  /datum/theft_objective/number/traitor/plasma_gas, needs to be restricted from the same people as TTVs.